catkin_ws/src/sensors/src/hydrophones_bearing.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+import rospy
+
+import numpy as np
+from std_msgs.msg import Float64MultiArray
+
+
+def cb_hydrophones(msg):
+    dt_h1 = msg.data[0]
+    dt_h2 = msg.data[1]
```

Comment:
could we make a custom message for hydrophones? Would make it easier to understand rather than just a float array